google/resumable_media/_download.py

```diff
@@ -58,6 +59,8 @@ def __init__(self, media_url, stream=None, start=None, end=None, headers=None):
         self.end = end
         if headers is None:
             headers = {}
+        version = pkg_resources.get_distribution('google-resumable-media').version
+        headers['User-Agent'] = "google-resumable-media-python/{}".format(version)
```

Comment:
can this be in a helper method to improve maintainability?

Reply:
Yep, we can do that.